src/charts/sparkline/sparkline-chart.directive.js

```diff
@@ -0,0 +1,327 @@
+/**
+ * @ngdoc directive
+ * @name patternfly.charts.directive:pfSparklineChart
+ *
+ * @description
+ *   Directive for rendering a sparkline chart.
+ *   <br><br>
+ *   See http://c3js.org/reference.html for a full list of C3 chart options.
+ *
+ * @param {object} config configuration settings for the sparkline chart:<br/>
+ * <ul style='list-style-type: none'>
+ * <li>.chartId            - the ID iof the container that the chart should bind to
```

Comment:
Yeah, that's all I do.  Go to the files tab and click the blue + on the line you want to comment on.